meshpy/mesh.py

```diff
@@ -779,6 +779,8 @@ def display_pyvista(
         beam_radius_for_display=0.1,
         resolution=20,
         is_testing=False,
+        parallel_projection=False,
+        add_cosys=False,
```

Comment:
I am fine with having this on by default, what do you think? @davidrudlstorfer

Reply:
I am totally fine with that. Do you want to enable it always (remove flags) or keep the flags and just set them to true?